xml/vnc.xml

```diff
@@ -980,4 +980,22 @@ WINDOWMANAGER=icewm vncserver -geometry 1024x768</screen>
    "Connection not encrypted!" message in that case).
   </para>
  </sect1>
+ <sect1 xml:id="sec-vnc-wayland">
+  <title>Compatibility with Wayland</title>
+  <para>
+    The Remote Admininstration (VNC) feature relies on X11 and may result in an
+    empty screen if Wayland is enabled.
+
+    The display manager must be configured to use X11 instead of Wayland.
+
+    For <package>gdm</package>, add <literal>WaylandEnable=false</literal> under
+    the <literal>[daemon]</literal> section to the
+    <filename>/etc/gdm/custom.conf</filename> configuration file.
```

Comment:
```suggestion
   For <package>gdm</package>, edit <filename>/etc/gdm/custom.conf</filename>.
   In the <literal>[daemon]</literal> section, add <literal>WaylandEnable=false</literal>
   to the configuration file.
```

Reply:
It is easier to follow the instructions when you first mention which file and which section to edit, and only then mention the parameter settings.